docs/developer/prepare-gcr-registry.md

```diff
@@ -63,17 +63,27 @@ Under the assumption you're [creating and using a service-account](https://kuber
    ```sh
    gcloud auth print-access-token --impersonate-service-account operator-test-sa@sap-kyma-jellyfish-dev.iam.gserviceaccount.com
 
-4. Verify your login:
+4. Adjust the `docker-push` command in `Makefile`
+   ```makefile
+   .PHONY: docker-push
+   docker-push: ## Push docker image with the manager.
+   ifneq (,$(GCR_DOCKER_PASSWORD))
+     docker login $(IMG_REGISTRY) -u oauth2accesstoken --password $(GCR_DOCKER_PASSWORD)
+   endif
+   docker push ${IMG}
+   ```
+
+5. Verify your login:
 
    ```sh
    gcloud auth print-access-token --impersonate-service-account operator-test-sa@sap-kyma-jellyfish-dev.iam.gserviceaccount.com | docker login -u oauth2accesstoken --password-stdin https://europe-west3-docker.pkg.dev/sap-kyma-jellyfish-dev/operator-test
    ```
-   export `GCR_DOCKER_PASSWORD` for `operator/docker-push` make command:
+   export `GCR_DOCKER_PASSWORD` for `docker-push` make command:
```

Comment:
`GCR_DOCKER_PASSWORD` is being set in step 5, but used in step 4. Add another step for setting the variable -> make file command -> kyma alpha module create